src/js/models/maps/viewfinder/ExpansionPanelsModel.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+define([], () => {
+  /**
+  * @class ExpansionPanelsModel
+  * @classdes ExpansionPanelsModel maintains state for multiple
+  * ExpansionPanelView instances so that only one is open at a time.
+  * @classcategory Models/Maps
+  */
+  const ExpansionPanelsModel = Backbone.Model.extend({
+    /**
+     * @name ExpansionPanelsModel#defaults
+     * @type {Object}
+     * @property {ExpansionPanelView[]} panels The expansion panel views that
+     * are meant to have only a single panel open at a time.
+     * @property {string} mode The open mode which determines the behavior of 
```

Comment:
would it be less typo prone to use a boolean instead? do we expect more than two modes?

Reply:
Done